test/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/LiabilityDetailsSpec.scala

```diff
@@ -67,18 +83,31 @@ class LiabilityDetailsSpec extends AnyWordSpec with Matchers {
 
     "be COMPLETED " when {
       "and 'isPreLaunch' flag is enabled" when {
-        "and liability details are all filled in" in {
+        "and liability details are all correctly filled in" in {
           val liabilityDetails =
-            LiabilityDetails(isLiable = Some(false), weight = Some(LiabilityWeight(Some(4000))))
+            LiabilityDetails(startDate =
+                               Some(Date(Some(1), Some(5), Some(2022))),
+                             expectedWeight =
+                               Some(LiabilityExpectedWeight(Some(true), Some(10000))),
```

Comment:
I was confused by this initially because I thought 10,000 was below the threshold but it's equal to the threshold. Is it worth defining some values for above and below threshold and using those in place of the specific values?

Reply:
Good idea